assets/src/edit-story/karma/fixture/containers/checklistPanel.js

```diff
@@ -20,10 +20,15 @@
 import { Container } from './container';
 
 /**
- * The editor's checklist.
+ * The editor's canvas. Includes: display, frames, editor layers, carousel,
+ * navigation buttons, page menu.
```

Comment:
That comment doesn't seem to match?

Reply:
sorry, this is a dev artifact :) I'll remove it